src/schema.prisma

```diff
@@ -1,6 +1,6 @@
 generator client {
   provider        = "prisma-client-js"
-  previewFeatures = ["referentialIntegrity", "fullTextSearch", "fullTextIndex"]
+  previewFeatures = ["fullTextSearch", "fullTextIndex"]
```

Comment:
would be nice to also add `relationJoins` feature which should make a perf difference

Reply:
@capaj done!

Reply:
@rafaell-lycan because relationJoins is still in preview are not enabled by default in findMany queries https://www.prisma.io/docs/orm/prisma-client/queries/relation-queries#relation-load-strategies-preview. So you need also to update all findMany queries